dating_workflow/step_script/get_gene_list_with_percentage.py

```diff
@@ -36,14 +36,13 @@ def cli(indir, ofile, suffix, num_percentage, num_genomes, genome_list):
         gids = f1.read().split('\n')
     gids = [convert_genome_ID(_) for _ in gids]
 
-    if num_genomes is None and num_percentage is None:
-        num_percentage = 100
+    if num_genomes is None:
         num_genomes = len(gids)
-    elif num_genomes is None:
-        num_percentage = int(num_percentage)
```

Comment:
should be kept since the passed params are string instead of number

Reply:
In the line 44, I used int(num_percentage) to solve the problem